foxpuppet/foxpuppet.py

```diff
@@ -8,7 +8,11 @@
 
 
 class FoxPuppet(object):
-    """ Class that sets up the api for interacting with the Firefox browser.
+    """
+        Class that sets up the interface for interacting with the Firefox
+        browser.
+
+        :param selenium: A selenium object
```

Comment:
Is there a good example of this within a docstring? It would work within an rst file but not within a py file.